File: srm181/kilo_man.py
class KiloMan:
	def hitsTaken(self, pattern, jumps):
		res = 0
		for i in range(len(pattern)):
			if (pattern[i] <= 2 and jumps[i] == 'S') or (pattern[i] > 2 and jumps[i] == 'J'): res += 1
		
		return res

File: srm181/KiloMan.java
public class KiloMan {

	public int hitsTaken(int[] pattern, String jumps) {
		int res = 0;
		for (int i = 0; i < pattern.length; i++) {
			if ((pattern[i] <= 2 && jumps.charAt(i) == 'S') || (pattern[i] > 2 && jumps.charAt(i) == 'J')) {
			 	res += 1;
			 }
		}		
		return res;
	}
}